src/Tinker.jl

```diff
@@ -126,6 +133,20 @@ function RectHandle(r::Rectangle)
     return RectHandle(r,h)
 end
 
+struct PolyHandle
+    p::Polygon
+    h::Tuple
+end
+
+function PolyHandle(p::Polygon)
+    # makes polyhandle
+    h = (Handle(p,1))
```

Comment:
These parentheses don't do anything:
```julia
julia> x = (5)
5

julia> typeof(x)
Int64
```
Did you mean to make a `tuple`? If so, add a comma, e.g., `h = (Handle(p, 1), )`. But note that `push!` doesn't work for tuples. I think here you might want `h = ntuple(i->Handle(p, i), length(p.pts))` or something.

But why are you creating this tuple at all? Not sure I follow the logic here. And if there are 1000 points, that's a long tuple (tuples are usually for short "lists", better to use an array for longer ones).